Resources/config/compile_dev.js

```diff
@@ -1,11 +1,13 @@
 {
-	"inherits": "./compile.js",
```

Comment:
One issue was that the whole file is re-indented. This made me reluctant to merge it, because it makes it harder for people to understand the history of the file if you let people move all the whitespace back and forth in each file according to their editor defaults. Also, it makes it hard to see which line has actually changed. I know this can seem really pedantic when somebody complains about your whitespace changes on a PR but it's important for keeping the project easy to maintain so I hope you're okay with it! 😁

Restoring the original whitespace would be the only _code change_ I'd ask you to make before merging this PR.